src/App.tsx

```diff
@@ -1,27 +1,41 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
```

Comment:
alternatively, you can optimize your todos arrays so that the request is not sent every time the button is clicked, for example:
`const allGodos = useMemo(() => getAll() ,[])`

and same for 5 first and red